tests/raftstore/test_snap.rs

```diff
@@ -288,7 +288,7 @@ struct StaleSnap {
 }
 
 impl Filter<RaftMessage> for Arc<StaleSnap> {
-    fn before(&self, msgs: &mut Vec<RaftMessage>) {
+    fn before(&self, msgs: &mut Vec<RaftMessage>) -> FilterResult {
```

Comment:
I am confused with FilterResult, and in fact, I don't think it is FilterResult, but only returns a result and let outer check, so why not using Result directly? And we can ignore checking msg.empty() too.